doc/version_history.rst

```diff
@@ -6,6 +6,16 @@
 Version History
 ##################
 
+.. _lsst.ts.m2gui-0.4.8:
+
+-------------
+0.4.8
```

Comment:
I think this should probably be a minor release as well, since there are some pretty substantial changes to the API.

Reply:
Done in the commit: [1345f96](https://github.com/lsst-ts/ts_m2gui/pull/44/commits/1345f9606637f278d8a022dd5e45ae808bc82220). Thanks!